packages/eas-cli/src/commands/channel/edit.ts

```diff
@@ -157,12 +154,21 @@ export default class ChannelEdit extends EasCommand {
       ({ name: branchName } = await promptAsync({
         type: 'text',
         name: 'name',
-        message: chalk`What branch should it change to?`,
+        message: chalk`To which branch should the channel link?`,
```

Comment:
Correct me if I'm wrong but I think the `chalk` tag doesn't have any effect here.
```suggestion
        message: `To which branch should the channel link?`,
```
I think someone writing the code meant `chalk.bold` but this wouldn't have had any effect too. AFAIR the prompt message is in bold either way.

Reply:
I unresolved the comment. It's still not addressed

Reply:
Ah, ok, I just realized you haven't pushed any commits yet. I'm ok with your workflow ;)